Sources/BraintreeSEPADirectDebit/BTSEPADirectDebitClient.swift

```diff
@@ -36,16 +43,29 @@ import BraintreeCore
             if error != nil {
                 completion(nil, error)
                 return
-            } else if result != nil {
-                // TODO: future PR start ASWebAuthenticationSession with result.approvalURL
+            } else if result != nil, let result = result {
+                if result.approvalURL == "null" {
+                    // TODO: call tokenize - url already approved
+                } else if let url = URL(string: result.approvalURL) {
+                    self.startAuthenticationSession(url: url, context: context) { success, error in
+                        switch success {
+                        case true:
+                            // TODO: call BTSEPADirectDebitClient.tokenize
+                            return
+                        case false:
+                            completion(nil, error)
+                            return
+                        }
+                    }
+                }
             }
         }
     }
     
     /// Initiates an `ASWebAuthenticationSession` to display a mandate to the user. Upon successful mandate creation, tokenizes the payment method and returns a result
     /// - Parameters:
     ///   - request: a BTSEPADebitRequest
-    /// - Note: This function should only be used for iOS 12 support.
+    /// - Note: This function should only be used for iOS 12 support. This function cannot be invoked on a device running iOS 13 or higher.
```

Comment:
I think there's a way to annotate this method as only being available up to and including iOS 12.

Reply:
I don't _think_ the `@available` tags allow for this. I know they are introducing an `@unavailable` tag in Swift 5.6 that would allow us to do that but don't think it exists with what we have currently.

Reply:
If you put @available above this function declaration, it will provide an error for the merchant trying to use it. I think that would probably be the ideal solve.